packaging/ubuntu-trusty/ecs.conf

```diff
@@ -0,0 +1,21 @@
+# Copyright 2014-2015 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the
+# "License"). You may not use this file except in compliance
+# with the License. A copy of the License is located at
+#
+#     http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+# CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and
+# limitations under the License.
+
+description "Amazon EC2 Container Service init"
+author "Amazon Web Services"
+start on (started docker and stopped cloud-final)
+
```

Comment:
I think adding *respawn* is a good idea to avoid throttling start in case of failure.